config/environments/development.rb

```diff
@@ -73,8 +73,11 @@
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
+  host = 'localhost:3000' # Local server
+  config.action_mailer.default_url_options = { host: host, protocol: 'http' }
```

Comment:
```suggestion
  config.action_mailer.default_url_options = { host: 'localhost:3000', protocol: 'http' }
```
prefer: code trực tiếp luôn thay vì đặt biến. vì mấy file config sẽ load cùng app nên hạn chế đặt biến (mà chỗ này đặt biến local cũng đc)